sa/_db/migrations/20151008132352_MoveOCSPResponseToCertificateStatus.sql

```diff
@@ -0,0 +1,23 @@
+
+-- +goose Up
+-- SQL in section 'Up' is executed when this migration is applied
+
+ALTER TABLE `certificateStatus` ADD COLUMN (`ocspResponse` mediumblob);
+UPDATE certificateStatus,ocspResponses SET certificateStatus.ocspResponse = ocspResponses.Response;
+DROP TABLE `ocspResponses`;
```

Comment:
I'm really worried about this. We have live production data in there right now.

I think we need a migration plan to fix this. Strawman: The first step is to add the data, but read from both the ocspResponses table and the certificateStatus. At the next deploy, we stop reading from ocspRespones. And then in the last, we can drop the table.